auto_install/install.sh

```diff
@@ -500,13 +571,14 @@ preconfigurePackages(){
 		[[ $WIREGUARD_BUILTIN == 1 && ( $PLAT == 'Debian' || $PLAT == 'Raspbian' ) ]] ||
 		# If the module is not builtin, on Raspbian we know the headers package: raspberrypi-kernel-headers
 		[[ $PLAT == 'Raspbian' ]] ||
+		# On Alpine, the kernel must be linux-lts or linux-virt if we want to load the kernel module
+		[[ "${PLAT}" == 'Alpine' && ! -f /.dockerenv && "$(uname -mrs)" =~ ^Linux\s+[0-9\.\-]+\-((lts)|(virt))\s+.*$ ]] ||
```

Comment:
This regex doesn't match, which means WireGuard is not available. It's due to `\s` not being supported by bash it seems. The following works unless there is some `uname` output where the space is not ASCII 32.

`[[ "${PLAT}" == 'Alpine' && ! -f /.dockerenv && "$(uname -mrs)" =~ ^Linux\ +[0-9\.\-]+\-((lts)|(virt))\ +.*$ ]]`